src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -1149,6 +1149,14 @@ void GraphOptimizer::FuseConvolutionAndSimpleOperation(Graph &graph) {
             continue;
         }
 
+#if defined(OV_CPU_WITH_ACL)
```

Comment:
Can we add more details why it is not working specifically for convolution node?

Reply:
@dmitry-gorokhov could you please propose the details I need to add here?